### PR TITLE
Aws shared kek

### PR DESCRIPTION
**Pull request by paoven**
- Created instructions to run CSFLE shared KMS flow
- Added KMS sharing toggle instructions as well
- Leveraging both Kotlin and CLI tools to test the feature
- All commands variable parameters can be configured through env variables

### CODE REVIEW

File: KafkaProducer/src/main/kotlin/KafkaProducer.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                try {
+                    kafkaProducer.send(ProducerRecord("$target_topic", personalData.getId(), personalData),
+                    ) { m: RecordMetadata, e: Exception? ->
+                        when (e) {
+                            null -> logger.info("event produced to ${m.topic()}")
+                            else -> logger.error("oh no, error occurred")
+                        }
+                    }
+                } catch ( e: SerializationException){
+                    ///logger.error("${e.cause} for $personalData")
```
Comment (pneff93) [inline KafkaProducer/src/main/kotlin/KafkaProducer.kt:41]: can be removed --> thanks, for the better logging :)

File: README.md [hunk 2]
```diff
@@ -28,5 +25,271 @@ We will produce personal data to Confluent Cloud in the following form
...
+
+```shell
+11:17:33.077 [Thread-0] INFO  KafkaProducer - Kafka Producer started
+11:17:34.495 [kafka-producer-network-thread | producer-1] INFO  KafkaProducer - event produced to csfle-kek-shared-demo
+11:17:36.665 [kafka-producer-network-thread | producer-1] INFO  KafkaProducer - event produced to csfle-kek-shared-demo
+```
+
+or check the encrypted field messages in the CC UI
+
+![](images/cc_ui_consume.png)
```
Comment (pneff93) [inline README.md:207]: I thought CC UI also has access here (as a client) and would decrypt it automatically

**Review by pneff93 — APPROVED** [review]
LGTM only one minor comment

File: README.md (hunk 2 above)
Comment (paoven) [reply README.md:207]: I think integration with CC UI/ksqlDB/Flink has not yet been developed. Not sure we are going to show unencrypted data in any case on CC UI, to be checked with the PM.

**Pull request merged**